### PR TITLE
Fix untranslated string

### LINKED ISSUE #305: Untranslatable stings
I found untranslatable strings. Can you make them translatable? I use red underscore for marking.

[Link to Imgur](https://imgur.com/a/6WRQRUa)

### PR DESCRIPTION
**Pull request by taooceros**
fix #305 
I fix most of them except for these two, which I am not sure how to add translated string to them.
image
image

### CODE REVIEW

File: Flow.Launcher/CustomQueryHotkeySetting.xaml [hunk 1]
```diff
@@ -19,22 +19,23 @@
...
+        <TextBlock Margin="38,0,0,0" FontSize="14" Grid.Row="1" Grid.Column="1" VerticalAlignment="Center"
+                   HorizontalAlignment="Left" Text="{DynamicResource actionKeyword}" />
+        <StackPanel Grid.Row="1" Orientation="Horizontal" Grid.Column="2">
             <TextBox x:Name="tbAction" Margin="10" Width="400" VerticalAlignment="Center" HorizontalAlignment="Left" />
             <Button x:Name="btnTestActionKeyword" Padding="10 5 10 5" Height="30" Click="BtnTestActionKeyword_OnClick"
                     Content="{DynamicResource preview}" />
         </StackPanel>
 
-        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Grid.Row="2" Grid.Column="1">
+        <StackPanel Orientation="Horizontal" HorizontalAlignment="Left" Grid.Row="1" Grid.Column="2" Margin="325,61,0,0" Grid.RowSpan="2">
```
Comment (jjw24) [inline Flow.Launcher/CustomQueryHotkeySetting.xaml:38]: is this change intentional? the button is not aligned. Can we revert this?

image

Comment (taooceros) [reply Flow.Launcher/CustomQueryHotkeySetting.xaml:38]: Sorry this seems unintentioned change, I will revert it.

**Review by jjw24 — APPROVED** [review]

**Comment by taooceros** [comment]
Thank you! I haven't got time to revert the change.😅

**Comment by jjw24** [comment]
All good, I was just doing review thought I might as well
